### DISCUSSION TITLE
Maska with Nuxt 3

### DISCUSSION THREAD

**Discussion by labbedaine** [beholdr/maska#196] [Q&A]
The website shows an example of how to setup Maska with Nuxt 3 but it uses an input instead of v-text-field (https://beholdr.github.io/maska/#/?id=usage).  

I am aware of the workaround (https://github.com/beholdr/maska/discussions/142) but I am looking for a solution that would allow me to set the mask on the v-text-field.

Any advice would be appreciated.

Thanks.

**Comment by PloggCTO** [comment]
You can to this
```
<script setup>
  const mask = ref({ mask: "####" });
</script>

<template>
  <v-input-field v-maska:[mask] />
</template>
```

Works for me.
